source/create.js

```diff
@@ -31,6 +35,24 @@ const create = defaults => {
 		handler: defaults.handler
 	});
 
+	got.merge = (...args) => {
+		const lastArgument = args[args.length - 1];
+		let methods;
+		let instances;
+
+		if (is.array(lastArgument)) {
+			methods = lastArgument;
+			instances = args.slice(0, -1);
+		} else {
+			methods = args[0].defaults.methods; // eslint-disable-line prefer-destructuring
```

Comment:
Is it even possible to do: `let x; {x} = {x: 123};`. Not yet 😕

Reply:
It is, but it's ugly...

```js
({methods} = args[0].defaults);
```

Reply:
I wonder why it's needed to enclose it in the brackets.

Reply:
From what I can remember, it's something about JS parsing ambiguity.